src/inverted-index.js

```diff
@@ -144,15 +139,14 @@ class InvertedIndex {
   }
 
   /**
-   * _stringify private method combines the title and
+   * _stringify private method merges the title and
    * text of each document into a single string
    *
    * @param {String} document
    * @return {String} stringified
    */
   _stringify(document) {
-    let stringified = document.title + " " + document.text;
-    return stringified;
+    return document.title + " " + document.text;
```

Comment:
Replace string concatenation with template literals since you're using es6 features
[https://github.com/airbnb/javascript#es6-template-literals](https://github.com/airbnb/javascript#es6-template-literals)